test/unit/instance/decrement.test.js

```diff
@@ -11,6 +11,13 @@ const sinon     = require('sinon');
 
 describe(Support.getTestDialectTeaser('Instance'), () => {
   describe('decrement', () => {
+    it('should disallow decrement if no primary key values is present', async () => {
+      const Model = current.define('User', {});
+      const instance = Model.build({}, { isNewRecord: false });
+
+      await expect(instance.decrement()).to.be.rejected;
+    });
```

Comment:
A few notes:

- don't name the model "Model", it's the name of the base model, prefer naming it after the table (e.g. "User").
- We're slowly rewriting our tests and we're dropping the "should" to reduce noise
- Add at least part of the error message to make sure it's the right error that arrises, and not an unrelated one
- following the previous note I think this should be two different tests:

```suggestion
    // this one will likely need to go in integration tests
    it('is disallowed if no primary key attribute is defined', async () => {
      const User = current.define('User', {}, { noPrimaryKey: true });
      const instance = await Model.create({});

      await expect(instance.decrement()).to.be.rejectedWith('part of the error message here (no need to copy all of it, just something that identifies it)');
    });
    
    // but this one can remain in unit tests
    it('is disallowed if the instance doesn't have its primary key', async () => {
      const User = current.define('User', {});
      // this instance was built but not saved, so it doesn't have a PK yet.
      const instance = Model.build({});

      await expect(instance.decrement()).to.be.rejectedWith('part of the error message here (no need to copy all of it, just something that identifies it)');
    });
```